app/build.gradle

```diff
@@ -1,14 +1,13 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 29
-    buildToolsVersion "29.0.3"
+    compileSdkVersion 30
```

Comment:
I got `Error: Google Play requires that apps target API level 30 or higher.` before this change, when I tried to generate a signed apk in Android Studio